webapp/lib/app/nook/view.dart

```diff
@@ -373,6 +375,16 @@ class ConversationPanelView with AutomaticSuggestionIndicator {
     _tags.children.removeWhere((Element d) => d.dataset["id"] == tagId);
   }
 
+  void markTagSelected(String tagId, bool selected) {
+    _tags.children.forEach((Element e) {
+      if (e.dataset["id"] == tagId) {
+        e.classes.toggle("tag--selected", selected);
+      } else {
+        e.classes.toggle("tag--selected", false);
+      }
+    });
```

Comment:
As per comment above, and also to match the intention of the method name, I think this can be simplified to:
```suggestion
    _tags.children
        .filterWhere((Element t) => t.dataset["id"] == tagId)
        .forEach((Element t) => t.classes.toggle("tag--selected", selected));
```
(I haven't run this code, so after accepting can you please check that it compiles and works etc.? thanks!)

Reply:
Nice - updated this and the below. Tested to make sure this works